cogboard-app/src/main/kotlin/com/cognifide/cogboard/config/helper/EntityCleanupHelper.kt

```diff
@@ -0,0 +1,67 @@
+package com.cognifide.cogboard.config.helper
+
+import io.vertx.core.Vertx
+import io.vertx.core.json.JsonArray
+import io.vertx.core.json.JsonObject
+import com.cognifide.cogboard.CogboardConstants as CC
+
+class EntityCleanupHelper(private val vertx: Vertx) {
+    fun handle(oldConfig: JsonObject, newConfig: JsonObject): List<String> {
+        return getWidgetIdsForPurge(oldConfig, newConfig)
+            .also { list -> list.forEach { item -> sendWidgetPurgeEvent(item) } }
+    }
+
+    private fun getWidgetIdsForPurge(oldConfig: JsonObject, newConfig: JsonObject): List<String> {
+        val oldBoards = getAllBoards(oldConfig)
+        val newBoards = getAllBoards(newConfig)
+
+        val allOldWidgets = getAllWidgets(oldBoards, oldConfig)
+
```

Comment:
You could remove this empty line

Reply:
I could do something a bit different and remove one of the variables, which will make this whole sequence 2 visual lines shorter.